easybuild/easyconfigs/c/CPC2/CPC2-0.1-foss-2022a.eb

```
@@ -0,0 +1,40 @@
+# Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+# Updated: Denis Kristak (Inuits)
+# Update to a Python3 version
+easyblock = 'PackedBinary'
+
+name = 'CPC2'
+version = '0.1'
+local_commit = '813487e'
+
+homepage = 'http://cpc2.cbi.pku.edu.cn'
```

```suggestion
homepage = 'http://cpc2.gao-lab.org/'
```
site claims to have migrated away from there, redirected here